### PR TITLE
Add support for debian 8

### PR DESCRIPTION
**Pull request by andrewschoen**
This adds support for debian 8. There were a few packages present in the packages list for debian 7 that are not present in debian 8.  I've just removed those for now and added a comment about them. I'm unsure how to determine if they're actually needed or not.

See: http://tracker.ceph.com/issues/12957

### CODE REVIEW

**Comment by dachary** [comment]
Verifying with
<pre>
teuthology-openstack --throttle 2 --verbose --key-name myself --key-filename ~/Downloads/myself --suite-branch wip-jessie-loic --ceph infernalis --suite dummy distros/all/debian_8.0.yaml debug/remove-me-ansible-jessie.yaml
</pre>

**Comment by andrewschoen** [comment]
@dachary great, thanks. I assume the ``remove-me-ansible-jessie.yaml`` file is pointing at this branch now?

File: roles/common/tasks/kerberos.yml [hunk 1]
```diff
@@ -7,6 +7,17 @@
     state: present
   when: ansible_distribution == 'RedHat'
 
+- name: Update apt cache.
```
Comment (zmc) [inline roles/common/tasks/kerberos.yml]: Shouldn't this go in ``main.yml``?

Comment (andrewschoen) [inline roles/common/tasks/kerberos.yml]: Maybe?  This was the only file that cared about apt at all in the common role, which is why I put it here.

I don't mind moving it if you think it should be though.

File: roles/testnode/vars/debian_8.yml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  - cifs-utils
+  ###
+  # DistCC for arm
+  - distcc
+  ###
+  # tgt
+  - tgt
+  - open-iscsi
+
+#NOTE: these packages were not found for debian 8, but are present for debian 7
```
Comment (andrewschoen) [inline roles/testnode/vars/debian_8.yml]: Anybody have context on these packages?  What we might be using them for or if we still need to be installing them?

**Comment by ktdreyer** [comment]
"distcc for ARM" sounds very obsolete

"nagios-nrpe-server" ... this allows a nagios server to contact a node and monitor it for alerts, no idea if we need it or not at this point

File: roles/common/tasks/kerberos.yml (hunk 1 above)
Comment (zmc) [inline roles/common/tasks/kerberos.yml]: Oh, I really thought apt was used elsewhere in the role. Don't care anymore :)

File: roles/testnode/vars/debian_8.yml (hunk 2 above)
Comment (zmc) [inline roles/testnode/vars/debian_8.yml]: @liewegas?

File: roles/common/tasks/kerberos.yml (hunk 1 above)
Comment (andrewschoen) [inline roles/common/tasks/kerberos.yml]: No biggie. :)

File: roles/testnode/vars/debian_8.yml (hunk 2 above)
Comment (liewegas) [inline roles/testnode/vars/debian_8.yml]: none of these are critical.

**Comment by dachary** [comment]
> I assume the remove-me-ansible-jessie.yaml file is pointing at this branch now?

Yes, that's all this ceph-qa-suite branch really has.

**Comment by dachary** [comment]
It failed with:
<pre>

**Comment by dachary** [comment]
Verifying with
<pre>
teuthology-openstack --throttle 2 --verbose --key-name myself --key-filename ~/Downloads/myself --suite-branch wip-jessie-loic --ceph infernalis --suite dummy distros/all/debian_8.0.yaml debug/remove-me-ansible-jessie.yaml
</pre>

* **fail** http://149.202.162.246:8081/ubuntu-2015-09-08_17:21:56-dummy-wip-kefu-testing---basic-openstack/
<pre>
{'target162146.teuthology': {'group': 'root', 'uid': 0, 'failed': True, 'state': 'file', 'gid': 0, 'mode': '0644', 'msg': 'chgrp failed: failed to look up group fuse', 'invocation': {'module_name': 'template', 'module_args': ''}, 'owner': 'root', 'path': '/etc/fuse.conf', 'size': 432}} 
</pre>

**Comment by andrewschoen** [comment]
@dachary It seems I lost some code in my last rebase.  I've added this commit that should fix that problem. https://github.com/ceph/ceph-cm-ansible/commit/4b5c1d5614071a4d440348b61c9863a654240fa5

**Comment by dachary** [comment]
Verifying with
<pre>
teuthology-openstack --throttle 2 --verbose --key-name myself --key-filename ~/Downloads/myself --suite-branch wip-jessie-loic --ceph wip-kefu-testing --suite dummy distros/all/debian_8.0.yaml debug/remove-me-ansible-jessie.yaml
</pre>

* **running** http://149.202.162.246:8081/ubuntu-2015-09-08_20:52:16-dummy-wip-kefu-testing---basic-openstack/

**Comment by dachary** [comment]
<pre>
Reviewed-by: Loic Dachary &lt;ldachary@redhat.com>
</pre>

**Pull request merged**